api/src/main/java/io/strimzi/api/kafka/model/JmxTransTemplate.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+package io.strimzi.api.kafka.model;
```

Comment:
Any reason why no to keep this in the `template` subpackage with the other template files?